improver/calibration/reliability_calibration.py

```diff
@@ -494,7 +494,7 @@ def process(self, cubes, coordinates=None):
         return result
 
 
-class ApplyReliabilityCalibration:
+class ApplyReliabilityCalibration(PostProcessingPlugin):
```

Comment:
Just as a query, I notice that the `AggregateReliabilityCalibrationTables` plugin isn't currently inheriting from anything. Should this plugin also be amended to inherit from `BasePlugin`?

Reply:
Added BasePlugin.